### PR TITLE
[move-prove] Update check PR script

### PR DESCRIPTION
**Pull request by tzakian**
This updates the `check_pr.sh` script to:
* Run unit tests for the Move stdlib and Diem Frameworks
* Generate the experimental artifacts for Diem Frameworks

## Testing
Ran it

### CODE REVIEW

File: language/move-prover/tools/check_pr.sh [hunk 1]
```diff
@@ -120,6 +124,13 @@ if [ ! -z "$GEN_ARTIFACTS" ]; then
       cd $dir
       cargo run
     )
+    if [[  $BUILD_EXPERIMENTAL = "$dir"  ]]; then
```
Comment (mengxu-fb) [inline language/move-prover/tools/check_pr.sh:127]:
```suggestion
    if [[  $BUILD_EXPERIMENTAL == "$dir"  ]]; then
```

Not an expert in bash though...

Comment (tzakian) [reply language/move-prover/tools/check_pr.sh]: Good spot! I got my `[[` and `[` commands mixed up... should be fixed now

**Review by wrwg — APPROVED** [review]

**Pull request closed without merging**